text/src/autogluon/text/automm/models/fusion.py

```diff
@@ -67,6 +67,7 @@ def __init__(
             Basically, it supports adding an auxilliary loss for each individual model.
         """
         super().__init__()
+        logger.debug("initializing MultimodalFusionMLP")
```

Comment:
Do we want to leave debug logger here?

Reply:
This is to be consistent with the debug loggers in other models' init functions. The debug messages are helpful to track whether code is run as expected in the debug mode. By default, debug message is not printed unless the `verbosity` hyper-parameter is set to 4.